compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -288,7 +312,11 @@ cmdBuild numProcessors =
     cmd =
         execBuild
             <$> projectOpts "daml build"
-            <*> optionsParser numProcessors (EnableScenarioService False) (pure Nothing)
+            <*> optionsParser
+                  numProcessors
+                  (EnableScenarioService False)
+                  (pure Nothing)
+                  disabledDlintUsageParser
```

Comment:
as in `cmdCompile`, in `cmdBuild` we don't allow any linting flags, but we could change that later easily